lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -330,6 +330,7 @@ def _section_data_download(course, access):
         'get_student_responses_url': reverse('get_student_responses', kwargs={'course_id': course_key.to_deprecated_string()}),
         'get_student_forums_usage_url': reverse('get_student_forums_usage', kwargs={'course_id': unicode(course_key)}),
         'get_ora2_responses_url': reverse('get_ora2_responses', kwargs={'course_id': course_key.to_deprecated_string()}),
+        'get_ora2_responses_url_email': reverse('get_ora2_responses_email', kwargs={'course_id': course_key.to_deprecated_string()}),
```

Comment:
It seems like we could avoid a lot of duplicated code if instead of creating a new endpoint, we parameterized the existing one with an include_email boolean.  Have you thought about doing that instead?  I may not be correctly understanding the extra information included in the email query, but it looks very similar to the original one.  I would much rather parameterize and reuse the existing ORA2 report generation methods than duplicate them.